modules/table/src/main/java/org/apache/ignite/internal/table/TableImpl.java

```diff
@@ -128,12 +122,12 @@ public int tableId() {
 
     @Override
     public void pkId(int pkId) {
-        this.pkId.complete(pkId);
+        this.pkId = pkId;
```

Comment:
Can we somehow init it from `InternalTable` in constructor? This setter makes no sense, `pkId` is known a priori.

Reply:
Why not, fix it.